app/views/hq/users/show.html.erb

```diff
@@ -18,7 +19,8 @@
   <%= pluralize(@user.inactive_sponsors.count.to_s, 'Inactive Sponsor') %>
   <% if @user.sponsors.count > 0 %>
     <div class="inactive_sponsors_index">
-      <%= render partial: 'hq/sponsors/sponsors.html.haml', locals: {sponsors: @user.inactive_sponsors} %>
+      <%= render partial: 'hq/sponsors/sponsors.html.haml',
+            locals: {sponsors: @user.inactive_sponsors.paginate(:page => params[:page])} %>
```

Comment:
If you use `params[:page]` it will lock both of them together; is there a solution involving something like `params[:inactive][:page]` but then that needs to be in the links in `_sponsors` ...